flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/sink/ElasticsearchSinkBaseITCase.java

```diff
@@ -165,8 +161,8 @@ private void runTest(
             @Nullable MapFunction<Long, Long> additionalMapper)
             throws Exception {
         final ElasticsearchSink<Tuple2<Integer, String>> sink =
-                ElasticsearchSink.builder()
-                        .setHosts(HttpHost.create(ES_CONTAINER.getHttpHostAddress()))
+                getSinkBuilder()
+                        .setHosts(HttpHost.create(getElasticsearchHttpHostAddress()))
```

Comment:
Can we move setting the host to `getSinkBuilder`? Then we have only one abstract method.

Reply:
No because we still need it for creating the RestHighlevelClient